tailcfg/tailcfg.go

```diff
@@ -830,6 +832,26 @@ type Hostinfo struct {
 	//       require changes to Hostinfo.Equal.
 }
 
+// VIPService represents a service created on a tailnet from the
+// perspective of a node providing that service. These services
+// have an IP address pair distinct from the node's IPs.
+type VIPService struct {
```

Comment:
Can you get the words "Virtual IP" into the comment somewhere so readers don't think this is a Very Important Person (Protocol?) and this is a VIP concierge service :)